harness/dotnet/Temporalio.Features.Harness/Runner.cs

```diff
@@ -64,15 +68,15 @@ public Task<WorkflowHandle> StartSingleParameterlessWorkflowAsync()
     {
         var workflow = WorkerOptions.Workflows.SingleOrDefault() ??
             throw new InvalidOperationException("Must have a single workflow");
-        return Client.StartWorkflowAsync(
-            workflow.Name!,
-            Array.Empty<object?>(),
-            new(id: $"{PreparedFeature.Dir}-{Guid.NewGuid()}", taskQueue: WorkerOptions.TaskQueue!)
-            {
-                ExecutionTimeout = TimeSpan.FromMinutes(1)
-            });
+        return Client.StartWorkflowAsync(workflow.Name!, Array.Empty<object?>(), NewWorkflowOptions());
     }
 
+    public WorkflowOptions NewWorkflowOptions() =>
+        new(id: $"{PreparedFeature.Dir}-{Guid.NewGuid()}", taskQueue: WorkerOptions.TaskQueue!)
+        {
+            ExecutionTimeout = TimeSpan.FromMinutes(1)
```

Comment:
Worth making this more aggressive? Presumably we don't actually want anything in this repo taking longer than a few seconds.

Reply:
:+1: May be worth suggesting in a more general place for visibility beyond the PR